### PR TITLE
Added blog posts for Meetup0 and Meetup1.

### PR DESCRIPTION
**Pull request by sidntrivedi**

### CODE REVIEW

**Comment by sidntrivedi** [comment]
@vaibhavk @ryzokuken @saranshbarua Please review.

File: src/pages/Open-source-M1/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: How to Open Source!
+date: "2018-08-20T23:46:37.121Z"
+---
+
+#### What is Open Source?
+#### How to start with Open Source?
+#### How to step into Open-Source Development?
+
+These are some of the simple yet daunting questions that many people have in mind. Some question up while the rest don't even care to understand it.And so, in order to remove this confusion or dilemma of the community members,a meetup was organized on the agenda
```
Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: These are some simple yet...

Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: Please add spaces after punctuation.

File: src/pages/Open-source-M1/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+#### What is Open Source?
+#### How to start with Open Source?
+#### How to step into Open-Source Development?
+
+These are some of the simple yet daunting questions that many people have in mind. Some question up while the rest don't even care to understand it.And so, in order to remove this confusion or dilemma of the community members,a meetup was organized on the agenda 
+"How to Open Source!" on 14th September,2018.
+
+An integral part of organizing meetups is to make the community members interact and network with each other.So,the meetup started with an introduction part where each member has to introduce himself/herself and the technology he/she is working on/interested to work on.
```
Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: Same thing about spaces after punctuation.

Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: Just use gender neutral language here.

> introduction part where participants had to introduce themselves and the technology they were working on/interested to work on.

File: src/pages/Open-source-M1/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+One can't start contributing to any open-source organization without knowing about the principles on which the philosophy of open-source is built upon.So having done with the introduction,talks were given based on the way to interact in an open-source community,introduction to open-source organizations,contemporary importance of open-source development etc.
+
+A successful strategy of open-source contributions which should be followed in order to start contributing to the organizations was also discussed according to which one has to start with the basics i.e. start with contributing to any typos/teensy errors in the Docs of an organization and then switch to pushing major changes.
```
Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: s/Docs/documentations

File: src/pages/Open-source-M1/index.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Further,a QnA round was held in which the community members were answered with the queries they had put forward regarding the the hurdles they faced while doing open-source contributions.
```
Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: s/Further/Finally

Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: > in which the community members asked questions regarding the the hurdles

File: src/pages/Open-source-M1/index.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+The meetup finally ended with the announcement of the next meetup regarding "Introduction to Git/Github" since the month long celebration of Open Source i.e. [Hacktoberfest](https://hacktoberfest.digitalocean.com/) has finally arrived.
```
Comment (ryzokuken) [inline src/pages/Open-source-M1/index.md]: s/Open Source/open-source software

File: src/pages/wanna-play-linux-meetup0/index.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+title: Wanna Play Linux Meetup
+date: "2018-09-20T23:46:37.121Z"
+---
+
+>One OS to rule them all,
```
Comment (ryzokuken) [inline src/pages/wanna-play-linux-meetup0/index.md]: A space between `>` and the sentence in this and all subsequent lines.

File: src/pages/wanna-play-linux-meetup0/index.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+>One OS to find them.
+>One OS to call them all,
+>And in salvation bind them.
+>In the bright land of Linux,
+>Where the hackers play.
+
+These magnificent lines by J.Scott Thayer themselves exhibit the power and superiority of Linux over other pieces of softwares or to be more precise,Kernels. Therefore, to provide the community members an introduction to the Linux command line and the various basic commands of Linux, OSDC organized the "Wanna-Play-Linux Meetup" on 30th August,2018.
```
Comment (ryzokuken) [inline src/pages/wanna-play-linux-meetup0/index.md]: Spaces after punctuation.

Comment (ryzokuken) [inline src/pages/wanna-play-linux-meetup0/index.md]: s/Kernels/kernels

File: src/pages/wanna-play-linux-meetup0/index.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+It started with an introductory talk by [Himanshu Nailwal](https://github.com/crapsystem) about the six stages of using Linux and some basic Linux commands. Adding to this,in order to make the learning process more interactive,a customised live disk was created which contained 11 levels, each of which contained a flag/password to the next level. The user had to find that flag using some basic commands like `cat` ,`ls -a`, `grep`, `find`,`tar` etc. in order to navigate his way to the next level and ultimately to the end.
+
+Bringing their hacking skills to the forefront, everyone and even the newcomers tried their best to decipher the flags level by level.This approach of learning by doing was highly appreciated by all the community members.
+
+This wargame simulation was followed by a QnA session in which the community members clarified their various doubts regarding using Linux and its Distros.The meetup finally ended with a lot of awesome stuff learnt and done.
```
Comment (ryzokuken) [inline src/pages/wanna-play-linux-meetup0/index.md]: s/Distros/distros

**Review by ryzokuken — CHANGES_REQUESTED** [review]
@sidntrivedi012 Great work! Just a couple of minor changes.

**Comment by sidntrivedi** [comment]
@ryzokuken Have made the changes. Please review.

**Review by ryzokuken — APPROVED** [review]
Looks great! Thanks.

**Pull request merged**